src/mo_frontend/parser.mly

```diff
@@ -607,7 +614,8 @@ exp_plain :
   | LPAR es=seplist(exp(ob), COMMA) RPAR
     { match es with [e] -> e | _ -> TupE(es) @? at $sloc }
 
-exp_nullary(B) :
+(* recovery comment: force to emit special variable instead of "_" to filter spurious errors *)
+exp_nullary [@recover.expr VarE ("__error_recovery_var__" @~ loc) @? loc] (B) :
```

Comment:
Instead of an unbound variable, that produces a spurious type error, we could insert the ast for 
  ``(primE "trap" : Text -> None) "..."``
or ``loop {}``, both of which have type `None`

Reply:
`prim "trap"` requires `import Prim "mo:⛔";` at the beginning of the file, doesn't it?

Reply:
The loop generates another spurious error: 
![image](https://github.com/user-attachments/assets/f5187ebe-8170-4dd6-99b1-c1746a3dd71a)